census.go

```diff
@@ -68,6 +68,8 @@ const (
 	FrameCensusTypeFollowers
 	// FrameCensusTypeFile is a census created from a file.
 	FrameCensusTypeFile
+	// FrameCensusTypeToken is a census created from the token holders of an NFT
+	FrameCensusTypeToken
```

Comment:
Shouldn't this be FrameCensusTypeNFT and FrameCensusTypeERC20 ?

Reply:
Yup changed